<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>406</width>
    <height>451</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>나눔고딕</family>
   </font>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#6495ED;
border-radius: 10px;

</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>70</y>
     <width>250</width>
     <height>198</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>Transparent_Logo2.png</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="ConnectVPN">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>340</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>50</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>맑은 고딕</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(47,18,241,0.3);</string>
   </property>
   <property name="text">
    <string>Connect VPN</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="FidoAuthentication">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>340</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>50</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>맑은 고딕</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(47,18,241,0.3);</string>
   </property>
   <property name="text">
    <string>FIDO Authentication</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
